include/accelerator.h

```diff
@@ -174,6 +174,58 @@ extern unsigned char detect_c128 (void);
  * 0x01  : C128 CPU is the current CPU
  */
 
+
+/* C64 Chameleon cartridge */
+
+extern unsigned char __fastcall__ set_chameleon_speed (unsigned char speed);
```

Comment:
Sorry for not noticing earlier :-( but we don't use "extern" for function declarations. Please remove them all here and in the comments section of the assembler source.
Additionally I ask you to remove the already merged ones - either in this pull request or a separate one, at your discretion.